wiki/installing-and-updating-drivers.md

```diff
@@ -0,0 +1,209 @@
+
+6. Windows will then restore your driver to the previous version, which could take up to 5-10 minutes.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/hardware-change-restart-prompt.png" alt="The prompt to restart your computer, shown after rolling back a driver." width="500px">
+
+
+Or if you can't access the normal safe mode, you may try and access Safe Mode with Command Prompt and execute the following commands in Command Prompt:
+
+1. ```Dism /image:C:/ /Get-Drivers```
+
+If you cannot access the normal safe mode, you may try and access Safe Mode with Command Prompt and execute the following commands in Command Prompt:
+
+3. You should see something like this:  
+```
+::: tip Note
+If you have trouble with this, proceed to another troubleshooting method below.
+:::
+```
+::: Info:
+In case something goes wrong during (operation), you can [proceed with another method of troubleshooting here](link to the thing) 
+:::
+
+1. At startup options, click Troubleshoot.
+
+   <img src="./img/installing-and-updating-drivers/winre/troubleshoot.png" alt="An image of advanced startup" height="350px">
+
+1. In the Windows Recovery Environment, click on Troubleshoot
+
+   <img src="./img/installing-and-updating-drivers/winre/advanced_options.png" alt="An image to entrace to advanced options" height="350px">
+
+2. Then, click on "Advanced Options"
+
+   <img src="./img/installing-and-updating-drivers/winre/system_restore.png" alt="An image to enter to system restore menu" height="350px">
+
+3. Finally, click on "System Restore" and follow the instructions
+
+<img src="./img/installing-and-updating-drivers/winre/system_restore.png" alt="An image to enter to system restore menu" height="350px">
+
+### Startup Repair
+    
+   1. Select "Troubleshoot"
+    
+2. Boot into the Recovery Drive and select "Fix Computer Problems"
+    
+   2. Select "Advanced options"
+    
+      <img src="./img/installing-and-updating-drivers/winre/advanced_options.png" alt="An image to entrace to advanced options" height="350px">
+    
+   3.  Select "Startup Repair"
+    
+      <img src="./img/installing-and-updating-drivers/winre/startup_repair.png" alt="An image to click startup repair" height="350px">
+    
+    - Select "Startup Repair"
+    
+    <img src="./img/installing-and-updating-drivers/winre/startup_repair.png" alt="An image to click startup repair" height="350px">
+
+### Reinstall Windows
+
+If none of the solutions above work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup. You may learn how to backup your data [here](/wiki/backup.html).
+
+1. Download Windows using a working computer: [Downloading Windows](/wiki/downloading-windows.html)
+ 
+If none of the solutions above work, you may need to reinstall Windows. This will erase all of your data and applications, so make sure you have a backup first. You may learn how to backup your data [here](/wiki/backup.html).
```

Comment:
```suggestion
If none of the solutions above work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup. You may learn how to backup your data [here](backup).

You can learn more on how to download Windows and install it [here](downloading-windows).
```